source/ncengine/graphics2/diligent/resource/component/ComponentResourceSignature.cpp

```diff
@@ -40,11 +42,13 @@ ComponentResourceSignature::ComponentResourceSignature(Diligent::IDeviceContext&
         throw NcError{fmt::format("Failed retrieving shader variable '{}'", meshRendererBufferVariableName)};
     }
 
-    m_meshRendererResource = std::make_unique<MeshRendererBufferResource>
+    m_meshRendererResource = std::make_unique<StructuredBuffer<MeshRendererData>>
     (
         context,
         device,
+        "MeshRendererStructuredBuffer",
```

Comment:
If it works, lets make this `meshRendererBufferVariableName` . It makes sense to me that the name you pass into `ShaderBindings.h` (variable name) should also be what appears in the GL/HLSL